CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## X.Y.Z 2024-XX-YY
+### PaymentSheet
+* [Fixed] PaymentSheet now uses a border width of 1.5 instead of 0 when `PaymentSheet.Appearance.borderWidth' is 0.
```

Comment:
Should we account for clear colors also?

Reply:
@porter-stripe https://github.com/stripe/stripe-ios/pull/3962/files#diff-61e0b8bf7fff7c04f9b42054f746789dcaa2365fb6273ff0cd810ade1daf6e24L53 we set border color to primary